reference_submissions/wmt/wmt_jax/submission.py

```diff
@@ -98,49 +96,32 @@ def init_optimizer_state(workload: spec.Workload,
 
 @functools.partial(
     jax.pmap,
-    in_axes=(None, None, 0, 0, 0, None, 0),
```

Comment:
lets revert all of the changes to this file, because we will need them later when we tune label smoothing in the v2 of the target setting runs (if we ever want to remove this code in the future, we can always git cherry-pick this commit).

Reply:
Maybe it makes sense to just add an additional `label_smoothing` argument to `workload.loss_fn()` if we want to tune label smoothing for all workloads anyway. Alternatively, we would have to allow one-hot labels as input to `workload.loss_fn()`; currently it says [Dense (not one-hot) labels](https://github.com/mlcommons/algorithmic-efficiency/blob/main/algorithmic_efficiency/spec.py#L225).

 In any case the submissions should use `workload.loss_fn()`, since it is part of the definition of a workload, i.e. if a user is not using it, their submission could be considered invalid.

Reply:
sgtm. I can add label_smoothing and dropout to the workload API in a follow-up PR as we discussed

Reply:
I wanted to suggest the same, I can add them in the target-setting run PR, since that is where they are needed anyway.